core/Config.php

```diff
@@ -138,6 +138,10 @@ public static function getDefaultLocalConfigPath()
      */
     public static function getLocalConfigPath()
     {
+        if (!empty($GLOBALS['CONFIG_PATH_RESOLVER']) && is_callable($GLOBALS['CONFIG_PATH_RESOLVER'])) {
+            return call_user_func($GLOBALS['CONFIG_PATH_RESOLVER']);
```

Comment:
is it secure to run a function defined in GLOBALS? I guess so, but wasn't sure...

Reply:
Yes it is secure. We use it also in various places.